src/Sylius/Bundle/AdminBundle/templates/shared/crud/common/content/form.html.twig

```diff
@@ -0,0 +1,23 @@
+{% import '@SyliusAdmin/shared/helper/button.html.twig' as _button %}
+
+{% if form is not defined %}
+    {% set form = hookable_metadata.context.form %}
+{% endif %}
+
+{% form_theme form '@SyliusAdmin/shared/form_theme.html.twig' %}
+
+<div class="container-xl" {% if attributes is defined %} {{ attributes }} {% endif %}> <!-- TODO: Remove the if statement once all resources will use the live component -->
```

Comment:
I would be for removing that comment because even if all resources will use live components, it is still good to leave this check in case developers in the end application want to use this template without creating a component, wdyt?
```suggestion
<div class="container-xl" {% if attributes is defined %} {{ attributes }} {% endif %}>
```